### PR TITLE
Updated Hindi language name and added Gujarati language.

### PR DESCRIPTION
**Pull request by winter-code**
1. Added Gu - Gujarati
2. Updated name_locale for Hi - Hindi.

### CODE REVIEW

File: django/conf/locale/__init__.py [hunk 1]
```diff
@@ -228,7 +234,7 @@
         'bidi': False,
         'code': 'hi',
         'name': 'Hindi',
-        'name_local': 'Hindi',
+        'name_local': 'हिंदी',
```
Comment (felixxm) [inline django/conf/locale/__init__.py]: @saurabhj Can you confirm?

**Comment by felixxm** [comment]
@winter-code Thanks. Changing local name for Hindi looks good, but I don't think that we have Gujarati translations (\cc @claudep).

File: django/conf/locale/__init__.py (hunk 1 above)
Comment (saurabhj) [reply django/conf/locale/__init__.py]: Yep. Looks good @felixxm

Comment (felixxm) [reply django/conf/locale/__init__.py]: Thanks!

**Comment by claudep** [comment]
@winter-code Do you intend to start a Gujarati translation for Django and take the role of Gujarati language coordinator? If you do, I can set it up on Transifex. Please give me your Transifex username.

**Comment by felixxm** [comment]
Merged Hindi language name in 4b45b6c8e4d7c9701a332e80d3b1c84209dc36e2.

**Comment by claudep** [comment]
Closing, as adding a new language requires Transifex setup first. @winter-code, my offer still stands.

**Pull request closed without merging**